esmgrids/cice_grid.py

```diff
@@ -78,6 +80,15 @@ def _create_2d_nc_var(self, f, name):
             complevel=1,
         )
 
+    def _create_3d_nc_var(self, f, name):
+        return f.createVariable(
+            name,
+            "f8",
+            dimensions=("nc", "nj", "ni"),
```

Comment:
```suggestion
            dimensions=("nj", "ni", "nvertices",),
```

Is the normal order / names for cice things

https://github.com/ACCESS-NRI/cice5/blob/a09f25ccebc43a4d47bb8b2bf1bd00f610062aae/io_netcdf/ice_history_write.F90#L474-L476

but maybe this order is needed for something else

Reply:
Ta, I just blindly copied the existing ESM1.6 grid. Changed in [0bd572e](https://github.com/COSIMA/esmgrids/pull/25/commits/0bd572e128432cc1c1c16bee52f16843f082d652)